package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "5-weather-app",
+  "version": "1.0.0",
+  "description": "You can learn the weather reports by typing in location.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch"
+  },
```

Comment:
* [ ] Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts`. "start" is missing. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))